src/Fantomas.Tests/StringTests.fs

```diff
@@ -339,3 +339,8 @@ let s =
     \"\"\"aaaa   
 bbb\"\"\"
 "
+
+[<Test>]
```

Comment:
I would keep it simple actually

```fsharp
[<Test>]
let ``unicode null character should be recognized as a trivia item, 2050`` () =
    formatSourceString false "let s = \"\\000\"" config
    |> should equal "let s = \"\\000\"\n"
```

Reply:
Thanks, the test looks better now and it actually passes only with the change in TokenParser.